buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -489,6 +490,7 @@ class BeamModulePlugin implements Plugin<Project> {
         grpc_protobuf                               : "io.grpc:grpc-protobuf:$grpc_version",
         grpc_protobuf_lite                          : "io.grpc:grpc-protobuf-lite:$grpc_version",
         grpc_netty                                  : "io.grpc:grpc-netty:$grpc_version",
+        grpc_netty_shaded                           : "io.grpc:grpc-netty-shaded:$grpc_version",
```

Comment:
Why do we need both grpc_netty and grpc_netty_shaded ?

Reply:
This comes from grpc-alts. (grpc-alts is for directpath)

Reply:
Note that current beam already has it from [gax-grpc](https://mvnrepository.com/artifact/com.google.api/gax-grpc/1.54.0) transitively. This can make sure that all these components are working with the same version.